service/tablet_allocator.cc

```diff
@@ -1189,7 +1189,9 @@ class tablet_allocator_impl : public tablet_allocator::impl
             auto tm = _db.get_shared_token_metadata().get();
             lblogger.debug("Creating tablets for {}.{} id={}", s.ks_name(), s.cf_name(), s.id());
             auto map = tablet_rs->allocate_tablets_for_new_table(s.shared_from_this(), tm, _config.initial_tablets_scale).get();
-            muts.emplace_back(tablet_map_to_mutation(map, s.id(), s.keypace_name(), s.cf_name(), ts).get());
+            tablet_map_to_mutations(map, s.id(), s.keypace_name(), s.cf_name(), ts, [&] (mutation m){
+                muts.emplace_back(std::move(m));
+            }).get();
```

Comment:
I think read_tablet_mutations() is also important to treat because that's when we read the whole tablet metadata.

Reply:
make sense. will add that as well.